### DISCUSSION TITLE
방학때 북클럽을 진행해보려합니다

### DISCUSSION THREAD

**Discussion by ing03201** [Soongsil-Developers/.github#18] [자유게시판]
진행은 리포지토리 하나 파서 각 장 별로 마크다운 파일 업로드하는것을 기본으로 하려합니다.
우선 생각하는 책은 네 가지입니다.
1. [클린코드](http://www.kyobobook.co.kr/product/detailViewKor.laf?mallGb=KOR&ejkGb=KOR&barcode=9788966260959)
2. [CSAPP](http://www.kyobobook.co.kr/product/detailViewKor.laf?mallGb=KOR&ejkGb=KOR&barcode=9791185475219)
3. [OSTEP](http://www.kyobobook.co.kr/product/detailViewKor.laf?mallGb=KOR&ejkGb=KOR&barcode=9791156007395) 
4. [코딩인터뷰 완전분석](http://www.yes24.com/Product/Goods/44305533)
5. [리팩터링](http://www.kyobobook.co.kr/product/detailViewKor.laf?ejkGb=KOR&mallGb=KOR&barcode=9791162242742&orderClick=LEa&Kc=)
공통기술서 1개, CS 책 1개, 알고리즘/코딩인터뷰 책 1개 이렇게 생각하고있습니다.
각 책당 4~6주로 생각하고있고 깃허브와 슬랙을 이용한 온라인으로 진행할 생각입니다
CS의 경우는 저자 lab에서 제공하는 homework 위주로 할거같습니다.
아직은 지원 받지않고 6월중순,말에 구글폼으로 받을예정입니다.
혹시나 다른 진행 방향을 원하시는 분은 말해주세요! 책 추천도 받습니다!
감사합니다

**Comment by enif-lee** [comment]
책 구성이 너무 좋네요. 와드박아두고 갑니다 ㅎㅎ

**Reply by ing03201** [reply]
감사합니다!

**Reply by ing03201** [reply]
코멘트 적어주셔서 대댓 달아요! ㅎㅎ 위에 수요조사 하고있는데 참여해주시면 감사하겠습니다
책 진행하게 된다면 대댓 달아드릴게요!

**Comment by mathpaul3** [comment]
너무 좋은 것 같습니다. 저도 와드 박아두고 갑니다!

**Reply by ing03201** [reply]
감사합니다!

**Reply by ing03201** [reply]
책 수요조사 글 올렸습니다!
모집글 올라오면 다시 대댓 달게요!

**Comment by LouiMinister** [comment]
저도 와드박구가요~~

**Reply by ing03201** [reply]
감사합니다! 모집글은 6월쯤에 다시 올릴게요!

**Comment by zwonlala** [comment]
저도 와드 박고 갑니다!

**Reply by ing03201** [reply]
감사합니다!

**Comment by acious** [comment]
이런 액티브한 활동 좋은거같아요 ㅎㅎ
감사합니다!

**Comment by seonghwan7694** [comment]
저도 와드 박고 갑니다!

**Comment by park-se-jun** [comment]
여유있으면 꼭 같이 하고 싶네요!

**Comment by rlaxowns7916** [comment]
저도 와드박고갑니다!
